app/lib/project/ImportController.js

```diff
@@ -216,18 +289,32 @@ define([
                 , kwargs: item.kwargs
             });
 
-            // this goes into the glyph
-            // returns an atNamespaceRule(penstroke:i({parentIndex})){ points ... }
-            rules.push(makeCPSPenStrokeRule(penStrokeData, parentIndex));
+            if(toMOM) {
```

Comment:
@graphicore: TODO: essentially the toMOM execution path is the future and importing to a CPS is obsolete. Thus, the latter should be removed.